### ISSUE TITLE
unexpected problem

### ISSUE THREAD

**Issue by HoxinhLuo** [spyder-ide/spyder#6426]
## Description

**What steps will reproduce the problem?**

1. 
2. 
3. 

**What is the expected output? What do you see instead?**

**Please provide any additional information below**

  File "C:\Python\Anaconda3\lib\site-packages\spyder\plugins\__init__.py", line 511, in <lambda>
    toggled=lambda checked: self.toggle_view(checked),
  File "C:\Python\Anaconda3\lib\site-packages\spyder\plugins\ipythonconsole.py", line 725, in toggle_view
    self.create_new_client(give_focus=False)
  File "C:\Python\Anaconda3\lib\site-packages\spyder\plugins\ipythonconsole.py", line 1033, in create_new_client
    self.connect_client_to_kernel(client)
  File "C:\Python\Anaconda3\lib\site-packages\spyder\plugins\ipythonconsole.py", line 1068, in connect_client_to_kernel
    kc.start_channels(shell=True, iopub=True)
  File "C:\Python\Anaconda3\lib\site-packages\qtconsole\kernel_mixins.py", line 43, in start_channels
    super(QtKernelClientMixin, self).start_channels(*args, **kw)
  File "C:\Python\Anaconda3\lib\site-packages\jupyter_client\threaded.py", line 212, in start_channels
    self.shell_channel._inspect = self._check_kernel_info_reply
  File "C:\Python\Anaconda3\lib\site-packages\jupyter_client\client.py", line 143, in shell_channel
    socket, self.session, self.ioloop
  File "C:\Python\Anaconda3\lib\site-packages\jupyter_client\threaded.py", line 205, in ioloop
    self._ioloop = ioloop.IOLoop()
  File "C:\Python\Anaconda3\lib\site-packages\tornado\util.py", line 312, in __new__
    instance.initialize(*args, **init_kwargs)
  File "C:\Python\Anaconda3\lib\site-packages\zmq\eventloop\ioloop.py", line 139, in initialize
    super(ZMQIOLoop, self).initialize(impl=impl, **kwargs)
  File "C:\Python\Anaconda3\lib\site-packages\tornado\ioloop.py", line 845, in initialize
    super(PollIOLoop, self).initialize(**kwargs)
  File "C:\Python\Anaconda3\lib\site-packages\tornado\ioloop.py", line 357, in initialize
    self.make_current()
  File "C:\Python\Anaconda3\lib\site-packages\tornado\ioloop.py", line 315, in make_current
    assert asyncio is None
AssertionError

## Version and main components

* Spyder Version: 3.2.6 
* Python Version: 3.6.4
* Qt Versions: 5.10.0, PyQt5 5.10 on Windows

## Dependencies
```
pyflakes >=0.6.0 :  1.6.0 (OK)
pycodestyle >=2.3:  2.3.1 (OK)
pygments >=2.0   :  2.2.0 (OK)
pandas >=0.13.1  :  0.22.0 (OK)
numpy >=1.7      :  1.14.0 (OK)
sphinx >=0.6.6   :  1.6.3 (OK)
rope >=0.9.4     :  0.10.5 (OK)
jedi >=0.9.0     :  0.10.2 (OK)
nbconvert >=4.0  :  5.3.1 (OK)
sympy >=0.7.3    :  1.1.1 (OK)
cython >=0.21    :  0.26.1 (OK)
qtconsole >=4.2.0:  4.3.1 (OK)
IPython >=4.0    :  6.2.1 (OK)
pylint >=0.25    :  1.7.4 (OK)

```

**Comment by ccordoba12** [comment]
Thanks for reporting. I have one question: is the Jupyter notebook working for you? Or is it failing as well?

**Comment by HoxinhLuo** [comment]
Yes! Jupyter Notebook works well.

**Comment by ccordoba12** [comment]
Then I don't know what the problem could be, because the error you reported is not related to Spyder.

Could you run in a system terminal (cmd.exe)

    jupyter qtconsole

and report back with any error? Thanks!

**Comment by CAM-Gerlach** [comment]
Also, particularly if the problem occurs on startup, please reference the [Spyder Troubleshooting Guide](https://github.com/spyder-ide/spyder/wiki/Troubleshooting-Guide-and-FAQ) for general information and fixes.

**Comment by CAM-Gerlach** [comment]
Closed due to lack of response. Please re-open if you can provide the requested information; thanks.

**Comment by ryanwiley2002** [comment]
CAM-Gerlach,

I experienced the same problem, but was able to fix it.  Although had to install Anaconda, as the problem attributed to my antivirus program blocking certain scripts and installation packages during the initial install.

**Comment by CAM-Gerlach** [comment]
So, the true source of the problem is now clear, thanks to some posters who provided more information with their reports. This is a duplicate of #6627 ; and in any case outside of Spyder's control. To fix for now, run ``conda install tornado=4.5.3`` from the Anaconda Prompt/Terminal/shell (Windows/Mac/Linux) in the ``conda`` environment that Spyder is installed in.

@ryanwiley2002 Glad you fixed your problem. I find it very unlikely (though not completely impossible) you had the same issue, as the above is caused by installing Tornado 5.0 instead of the default 4.x; we've had several different startup issues that indeed are caused by antivirus programs doing what you described.
